packages/pieces/community/matomo/src/lib/api.ts

```diff
@@ -7,20 +7,29 @@ import {
 import { MatomoAuthType } from './auth';
 
 const matomoAPI = async (
-  api: string,
-  auth: MatomoAuthType,
-  queryParams: QueryParams = {}
+  api         : string,
+  auth        : MatomoAuthType,
+  queryParams : QueryParams = {}
 ) => {
-  queryParams['module'] = 'API';
-  queryParams['format'] = 'JSON';
-  queryParams['method'] = api;
-  queryParams['token_auth'] = auth.tokenAuth;
-  queryParams['idSite'] = auth.siteId;
+	queryParams['module'] = 'API';
+	queryParams['format'] = 'JSON';
+	queryParams['method'] = api;
+	queryParams['idSite'] = auth.siteId;
+
+  const formData = new FormData();
+  formData.append('token_auth', auth.tokenAuth);
+
+  console.log('queryParams', queryParams);
```

Comment:
Hey @joeworkman, can you please remove these log messages? It contains auth details.

Reply:
Doh! Sorry. I'll get those removed.